frontend/packages/topology/src/components/quick-search/QuickSearchList.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import cx from 'classnames';
+import { history, removeQueryArgument } from '@console/internal/components/utils';
+import { CatalogItem } from '@console/plugin-sdk';
+import {
+  DataList,
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells,
+  DataListCell,
+  Split,
+  SplitItem,
+  Label,
+  TextContent,
+  Text,
+  TextVariants,
+  Button,
+} from '@patternfly/react-core';
+import './QuickSearchList.scss';
+
+interface QuickSearchListProps {
+  listItems: CatalogItem[];
+  totalItems: number;
+  selectedItemId: string;
+  searchTerm: string;
+  namespace: string;
+  onSelectListItem: (itemId: string) => void;
+}
+
+const QuickSearchList: React.FC<QuickSearchListProps> = ({
+  listItems,
+  totalItems,
+  selectedItemId,
+  searchTerm,
+  namespace,
+  onSelectListItem,
+}) => {
+  const { t } = useTranslation();
+  const ref = React.useRef(null);
+
+  const openForm = (e: React.SyntheticEvent) => {
+    e.persist();
+    e.stopPropagation();
+    e.preventDefault();
+    const targetItemUID = ref?.current?.props?.id;
```

Comment:
```suggestion
    const targetItemUID = ref.current?.props?.id;
```
Pretty sure ref ALWAYS has `current` as it's the structure of the `useRef` 🤔